sqrl-openai/openai-cicd-test.sqrl

```diff
@@ -1,11 +1,14 @@
042490987, -0.015709816, -0.00582008, 0.012202193, -0.018689264, 0.025257591, -0.005363006, 0.004800974, -0.022589631, -0.012872568, 0.0036125805, -0.022183342, -0.011890705, -0.025108619, -0.01246628, -7.465548E-4, -0.011294816, 0.005965667, -0.0067173005, -0.0069069015, 0.021506196, -0.024106441, 0.0053156056, 0.01848612, 0.0053562345, -0.020490475, -0.038759906, 0.016414048, -0.008687799, -0.016752621, 0.002512216, -0.018242346, -0.0151545545, -0.0015422028, -0.036701377, 0.022521917, 0.01182299, -0.01637342, 0.015032668, -0.00855237, -0.0019298695, -0.012622024, 0.0024665087, 0.01172819, -0.016427591, 0.01164016, -0.02520342, 2.6472224E-4, -0.023659524, -0.02101865, 0.009452975, -0.023998097, 0.013096027, 0.01766, 0.010069179, 0.02764115, 0.021817682, 0.008660713, 0.0026713458, -0.008538826, -0.0043811426, 0.031013342, -0.026611885, -0.023984555, -0.015195183, 0.007929394, 0.012737139, 0.00538332, 0.010082722, -0.024201242, 0.010685383, 0.003333257, 0.031663403, 0.0037006096, -0.010976557, -0.030065335, -0.008295054, -0.011389616, -0.01710474, 0.013908604, 0.0072048465, -0.0051226187, 0.0080986805, 0.008565912, 0.022914661, 0.013854432, 0.011958419, -0.026625428, -0.006090939, -0.020680076, 0.010001465, 0.004415, 0.007414762, -0.015994217, 0.022278143, 0.002028056, 0.023158435, -0.0012721904, 1.4759693E-4, 0.0054476494, -0.00874197, 0.0046757017, 0.002320922, -0.04534178, 0.0077059353, 0.015533757, 0.021682253, 0.0021600996, -0.0028694114, 0.005092147, -0.010407752, 0.004259256, -0.0015938353, 0.025325306, -0.009940521, -0.0058810236, -0.013800261, 0.014301349, 0.020355046, 0.025975367, -1.8589807E-4, 0.0068121008, 0.029713219, -0.001950184, -0.01669845, 0.004936403, -0.019122638, 0.0018113687, 0.016116103, 0.023510551, -0.018756978, 0.0037683242, 0.031284202, 0.026760858, 0.004100126, 0.0073538185, -0.0053156056, -0.01692868, -0.0034466793, -0.013732545, -0.012520452, -0.029550705, -0.0034551437, 0.033559415, 0.018459033, -0.017009938, -0.024296042, 0.012655881, -0.0053122197, 0.016657822, -0.0154524995, 0.004056112, -0.018526748, -0.0111323, 0.013075713, -0.004848374, 0.0035347084, 0.012506909, -0.011626618, 0.029848648, 0.011741732, -0.014788895, -0.007888765, -0.025867024, 5.451035E-4, 0.004987189, 0.006476913, -0.011369302, -0.012682967, 0.005423949, -0.02042276, 0.0078413645, -0.026097253, 0.04236233, 0.00929046, 0.0028761828, -0.010692154, -0.0014347057, 0.018350689, 0.027763035, 0.017009938, -0.020409217, -0.00310472, 0.009439432, -0.009412346, -0.009446204, -0.033559415, -0.012046449, 0.02654417, -0.0039985543, 0.00713036, -0.010082722, -0.019339325, 0.029875735, 0.00333495, 0.02575868, -0.01357003, -0.012026135, -0.01798503, 0.018662177, 0.0010360352, 0.0029269687, -0.02846727, 0.0043438994, -0.011924562, -0.022142714, -0.0051632477, -0.012615252, 0.0061755828, -0.002695046, -0.024119984, 0.009513918, 0.015682729, -0.019501839, -5.920806E-4, -0.016075475, 0.013366886, 0.007922622, 0.011342215, 0.008599769, -0.018824693, -0.01863509]
```

Comment:
It’s inline deliberately to keep everything in one place, but I can see the value in moving it to a separate file. Let me know your thoughts.